WordPress/Classes/Networking/BlogServiceRemoteREST.m

```diff
@@ -217,6 +252,15 @@ - (NSDictionary *)mapPostFormatsFromResponse:(id)response
     }
 }
 
+- (RemotePostType *)remotePostTypeWithDictionary:(NSDictionary *)json
+{
+    RemotePostType *postType = [[RemotePostType alloc] init];
```

Comment:
Do you mean to catch a bad dictionary that wouldn't resolve to a `RemotePostType`?